### PR TITLE
Don't trigger log or lg without brackets (as these don't work)

### LINKED ISSUE #4246: Calculator: No result but calculator opens
## Steps to recreate
https://duckduckgo.com/?q=log+0.000000000000001&t=vivaldi&ia=calculator

## People to notify

---

Instant Answer Page: https://duck.co/ia/view/calculator

### PR DESCRIPTION
**Pull request by kirkins**
## Description of new Instant Answer, or changes

This stops `log X` and `lg X` from being triggered on search, as said formats don't work.

## Related Issues and Discussions

Fixes #4246

## Testing & Review
To be completed by Community Leader (or DDG Staff) when reviewing Pull Request

**Pull Request**
- [ ] Title follows correct format (Specifies Instant Answer + Purpose)
- [ ] Description contains a valid Instant Answer Page Link (e.g. https://duck.co/ia/view/my_ia)

**Instant Answer Page** (for new Instant Answers)
- [ ] Instant Answer page is correctly filled out and contains:
    - The **Title** is appropriately named and formatted
    - The **IA topics** are present and appropriate
    - The **Description** is clear and coherent
    - **Source Name** exists if applicable
    - All **Example Queries** trigger on [Beta](https://beta.duckduckgo.com/)

**Code**
- [ ] Adheres to the [DuckDuckGo Style Guide](https://docs.duckduckhack.com/resources/code-style-guide.html)
- [ ] Behaviour is appropriately tested. If improvement, tests are adequately extended.
- [ ] There is no unnecessary files in place (such as editor config files)
- [ ] There is no API keys / secrets present
- [ ] Tests are passing (run `$ duckpan test <goodie_id>`)
    - Tester should report any failures

**Ready to merge?**

- [ ] Has this IA been deployed to and tested on [beta.duckduckgo.com](https://beta.duckduckgo.com/)?
- [ ] For larger commits, has this been approved be more than one community member?
- [ ] The number of reviews is appropriate for this type of PR
- [ ] The commit message is clear, coherent and fitting

**Pull Request Review Guidelines**: https://docs.duckduckhack.com/programming-mission/pr-review.html

---

Instant Answer Page: https://duck.co/ia/view/calculator

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: t/Calculator.t [hunk 1]
```diff
@@ -307,7 +307,7 @@ ddg_goodie_test(
     '323.7+ 55.3%' => build_test(
         '323.7 + 55.3%'
     ),
-    '577.40*5%' => build_test(
+    '577.40*5%' => build_test(f)r
```
Comment (kirkins) [inline t/Calculator.t]: My bad, using the web browser to make changes.

**Comment by kirkins** [comment]
Sorry for so many commits. First I accidentally removed the wrong test. 

Then I realized the problem described in issue  #4246 also applies to `ln X`.

So I stopped this from triggering as well and then removed the associated tests.

**Comment by bsstoner** [comment]
This was released to production 🚀

**Pull request merged**